department-of-veterans-affairs/caseflow
# Add Assignee Label to CachedAppealAttributes for Legacy Appeals

Bump #11316 

### Description
Caches the Appeal location for Legacy Appeals in the CachedAttributesTable

- [x] VACOLs locations cached
- [x] Caseflow locations cached

### Acceptance Criteria
- [ ] Job caches Assignee Label correctly for each Legacy Appeal

### Testing Plan
1. Run the UpdateCachedAppealAttributes job
1. Confirm all Legacy appeals in the DB have a correct assignee_label
   1. Nothing should have the 'CASEFLOW' location
